packages/client/__tests__/AuthenticationPasswordReset.test.tsx

```diff
@@ -335,8 +338,55 @@ const passwordResetMachine =
                                 },
                             },
                         },
-                        'Token validated': {
-                            type: 'final',
+                        'Requested password reset code resending': {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Really nice !